librad/src/git/types/remote.rs

```diff
@@ -427,7 +427,7 @@ impl Remote<LocalUrl> {
                 specs,
                 Some(
                     git2::FetchOptions::new()
-                        .prune(git2::FetchPrune::On)
+                        .prune(git2::FetchPrune::Off)
```

Comment:
Is this bool flip intentionally part of this PR?

Reply:
Nope, reverted.